src/hooks/useDarkMode.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+import { useLocalStorage } from './useLocalStorage'
+
+export function useDarkMode() {
+    const [value, setValue] = useLocalStorage('darkMode');
+
+    useEffect(() => {
+        const body = window.document.body;
+        if (value) {
+            body.classList.add('dark-mode');
+        } else {
+            body.classList.remove('dark-mode');
+        }
+    }, [value]);
```

Comment:
I know the instructions say to modify the dom directly, but it's not very react like to do that.  How would you solve the problem otherwise?